### PR TITLE
Added a few more files to diagnostics bundle for support

### PR DESCRIPTION
**Pull request by surdy**
## High Level Description

What features does this change enable? What bugs does this change fix? High-Level description of how things changed.
This change adds a few more files from master node to a diagnostics bundle.These files come in handy for support debugging various issues

Enterprise PR https://github.com/mesosphere/dcos-enterprise/pull/576

## Related Issues

  - [DCOS-13706](https://mesosphere.atlassian.net/browse/DCOS-13706) 

## Checklist for all PR's

  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [X] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [X] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and Review Process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using github's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by surdy** [comment]
Updated the PR to remove zookeeper.out (Ref https://github.com/dcos/3dt/pull/84#issuecomment-278270815)

**Comment by surdy** [comment]
@darkonie Can you please review this ?

**Comment by darkonie** [comment]
@surdy can you please open another PR for dcos-enterprise?

**Comment by surdy** [comment]
@darkonie I was waiting for this one to be reviewed so that I do not repeat the mistakes in both PRs if I make any. 

I will go ahead an create the enterprise PR as well and let you know

**Comment by darkonie** [comment]
🚢 when enterprise PR is ready. Also please paste a link to enterprise PR in the description.

**Comment by surdy** [comment]
Added the link to enterprise PR in description

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -698,6 +698,21 @@ package:
           "LocalFiles": [
               {
                   "Location": "/opt/mesosphere/active.buildinfo.full.json"
+              },
+              {
+                  "Location": "/opt/mesosphere/active/dcos-metadata/etc/dcos-version.json"
```
Comment (branden) [inline gen/dcos-config.yaml]: Is there a reason we shouldn't use `/opt/mesosphere/etc/dcos-version.json` instead? If it's all the same, we might as well use the path that won't break if we change the name of the package that provides this file.

Comment (surdy) [inline gen/dcos-config.yaml]: I just did not realize this information existed in `/opt/mesosphere/etc/dcos-version.json` as well. I'll update the PR

**Comment by surdy** [comment]
@darkonie is there a way to specify "get these files on Masters only"
`/var/lib/dcos/exhibitor/zookeeper/snapshot/myid` and `/var/lib/dcos/exhibitor/conf/zoo.cfg` would not exist on agents.

**Comment by darkonie** [comment]
@surdy yes, you can add here https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/test_3dt.py#L532 and/or https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/test_3dt.py#L535

**Comment by surdy** [comment]
@darkonie My question is not about the integration test, but the actual collection of files from the nodes. 
When I specify files in `LocalFiles` is there a way to say this file if only to be collected from masters 

`/var/lib/dcos/exhibitor/zookeeper/snapshot/myid` and `/var/lib/dcos/exhibitor/conf/zoo.cfg` would not be available on agents

**Comment by darkonie** [comment]
@surdy ah sorry, yes you can add `"Role": ["master"]`

see example here https://github.com/dcos/dcos/blob/master/gen/dcos-config.yaml#L706
this should work for files as well.

**Comment by surdy** [comment]
Updated to include the files in integration test

**Comment by surdy** [comment]
@darkonie I made a couple of changes to make tests pass. Please review again. I have also updated https://github.com/mesosphere/dcos-enterprise/pull/576. But the enterprise repo does not seem to have `test_3dt.py` is that expected ?

**Comment by darkonie** [comment]
@surdy looks good to me! Thanks

**Comment by surdy** [comment]
@darkonie @branden Can you please review and approve ?

**Comment by darkonie** [comment]
@surdy you git approval from me 🚢

**Comment by surdy** [comment]
Oddly this still says review required. Do I need to do anything else to unblock this merge ?

**Comment by branden** [comment]
It says review required because it hasn't been approved by someone with write permission on this repo yet. Now that it has two approvals and no outstanding rejections, it'll get pulled into the next merge train, and that train will eventually be approved by someone with write permission and merged.

**Comment by orsenthil** [comment]
@surdy - even this needs rebase + squash.

**Pull request merged**